DigitalLearningSolutions.Data/Services/CourseContentService.cs

```diff
@@ -57,24 +72,26 @@ public CourseContentService(IDbConnection connection, ILogger<CourseContentServi
                             THEN 0
                             ELSE CAST(SUM(aspProgress.TutStat) * 100 AS FLOAT) / (COUNT(Tutorials.TutorialID) * 2)
                          END) AS PercentComplete
-                  FROM Applications

```

Comment:
From here downwards it's just spacing being changed right? No functional change? 🙂

Reply:
Yeah that's right, I didn't spot that I had changed that but I think it's probably better 🙂